src/tests/__snapshots__/CompleteOptionsToggles.test.tsx.snap

```diff
@@ -45,7 +45,7 @@ exports[`CompleteOptionsToggles matches snapshot 1`] = `
               disabled=""
             /><span class="switch switch-background"
               ><span class="switch switch-button"></span></span
-            >Essential purposes</label
+            >Essential</label
```

Comment:
@csmccarthy not sure why this and yarn files updated

Reply:
i think we just forgot to run the snapshot update when going through the message changes. a diff here is nothing to worry about bc these strings will never be shown to the end user since purpose messages will always be defined